src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,29 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = null;
         try {
             user = authenticationService.login("bob", "1234");
+            logger.info("User {} logged in successfully.", getUserLogin(user));
```

Comment:
The logger.info() statement uses a method getUserLogin(user) that could return an empty string, this could lead to confusion when analyzing logs as it would not be clear which user the log refers to.